src/components/WithdrawPage.tsx

```diff
@@ -85,99 +85,109 @@ const WithdrawPage = (props: Props): ReactElement => {
+                      value: "ALL",
+                    })
+                  }
+                  label="Combo"
+                />
+                {tokensData.map((t) => {
+                  return (
+                    <RadioButton
+                      key={t.symbol}
+                      checked={formStateData.withdrawType === t.symbol}
+                      onChange={(): void =>
+                        onFormChange({
+                          fieldName: "withdrawType",
+                          value: t.symbol,
+                        })
+                      }
+                      disabled={poolData?.isPaused}
+                      label={t.name}
+                    />
+                  )
+                })}
+              </div>
+              {tokensData.map((token, index) => (
+                <div key={index}>
+                  <TokenInput
+                    {...token}
+                    // inputValue={parseFloat(token.inputValue).toFixed(5)}
+                    onChange={(value): void =>
+                      onFormChange({
+                        fieldName: "tokenInputs",
+                        value: value,
+                        tokenSymbol: token.symbol,
+                      })
+                    }
+                    disabled={poolData?.isPaused}
+                  />
+                  {index === tokensData.length - 1 ? (
+                    ""
                   ) : (
-                    <span className="slippage">{t("priceImpact")}</span>
+                    <div className="formSpace"></div>
                   )}
-                  <span
-                    className={
-                      "value " +
-                      (reviewData.priceImpact.gte(0) ? "bonus" : "slippage")
-                    }
-                  >
-                    {formatBNToPercentString(reviewData.priceImpact, 18, 4)}
-                  </span>
+                </div>
+              ))}
+              <div className={"transactionInfoContainer"}>
+                <div className="transactionInfo">
+                  <div className="transactionInfoItem">
+                    {reviewData.priceImpact.gte(0) ? (
+                      <span className="bonus">{t("bonus")}: </span>
+                    ) : (
+                      <span className="slippage">{t("priceImpact")}</span>
+                    )}
+                    <span
+                      className={
+                        "value " +
+                        (reviewData.priceImpact.gte(0) ? "bonus" : "slippage")
+                      }
+                    >
+                      {formatBNToPercentString(reviewData.priceImpact, 18, 4)}
+                    </span>
+                  </div>
                 </div>
               </div>
             </div>
-          </div>
-          <AdvancedOptions />
-
-          <Stack direction="row" width="434px" pt={3} justifyContent="center">
+            <Box px={[3, 3, 0]} width="100%">
```

Comment:
This is good example of giving padding according to the break point in MUI.
For more detail, https://mui.com/system/basics/#responsive-values

Reply:
Are breakpoints necessary to work on mobile or can pure Flexbox/Grid handle it?

Reply:
Pure Flexbox/Grid can handle it.
But for now the margin of the token Input is handled by scss, so now I gave padding according to the breakpoint as temporally.